packages/react/src/utils/useScrollLock.ts

```diff
@@ -85,18 +83,23 @@ function preventScrollStandard(referenceElement: Element | null) {
     const scrollbarWidth = Math.max(0, win.innerWidth - html.clientWidth);
     const scrollbarHeight = Math.max(0, win.innerHeight - html.clientHeight);
 
+    // Avoid shift due to the default <body> margin. This does cause elements to be clipped
+    // with whitespace. Warn if <body> has margins?
+    const marginY = parseFloat(bodyStyles.marginTop) + parseFloat(bodyStyles.marginBottom);
+    const marginX = parseFloat(bodyStyles.marginLeft) + parseFloat(bodyStyles.marginRight);
```

Comment:
`bodyStyles` is a computed styles object, which is [live](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle#return_value).